src/include/btmem.h

```diff
@@ -911,8 +853,7 @@ struct __wt_ref {
 #undef ref_ikey
 #define ref_ikey key.ikey
 
-    WT_PAGE_DELETED *page_del;   /* Deleted page information */
-    WT_PAGE_LOOKASIDE *page_las; /* Lookaside information */
+    WT_PAGE_DELETED *page_del; /* Deleted page information */
```

Comment:
`WT_REF_SIZE` needs an update its size according to the the `__wt_ref` update.

Reply:
d20f19e